pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -180,7 +180,6 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		// for now this is what we want. in the future we may revisit this.
 		"--privileged",
 		"--security-opt", "seccomp=unconfined", // also ignore seccomp
-		"--security-opt", "apparmor=unconfined", // also ignore apparmor
```

Comment:
is there any side effect removing this?

Reply:
ic 
```
Command Output: Error: invalid config provided: AppArmorProfile and privileged are mutually exclusive options
```
I'm not very much into apparmor, so my question is about the user experience, does this means the users need to disable apparmor or use specific profiles, or just will work?

Reply:
Let's just specify `--privileged` and drop all `--security-opt *`.

Reply:
> Let's just specify `--privileged` and drop all `--security-opt *`.

that is better, thanks for the suggestion.  I've updated the PR

Reply:
how does this apply to older versions of podman?

Reply:
were these flags just ignored? docker does not throw this error.

Reply:
I think it is just a regression in Podman that raises an error when both `--privileged` and `--security-opt apparmor=*` are used.  There is already an issue to track the problem in podman 2.0 so that `--security-opt apparmor=` overrides `--privileged`.  For our use case though it doesn't matter since `--privileged` implies no apparmor and selinux

Reply:
SGTM, I just want to make sure we're not not-reporting these :sweat_smile: